project_milestone_time_kpi/views/project_project.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="project_form" model="ir.ui.view">
+        <field name="name">Project: add milestone time kpi</field>
+        <field name="model">project.project</field>
+        <field name="inherit_id" ref="project_milestone_enhanced.project_active_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='milestones']" position="inside">
+                <group name="milestone_time_kpi" class="oe_subtotal_footer oe_right" colspan="2">
+                    <field name="total_estimated_hours" widget="float_time" />
```

Comment:
Hello @lanto-razafindrabe the widget "float_time" does'nt show the hours in right format 
Plz replace it by widget="timesheet_uom"
Add also sum="FIELD_NAME" optional="show" groups="hr_timesheet.group_hr_timesheet_user"

Same thing for following fields
Change also Readme pictures